serverlessworkflow/modules/ROOT/pages/cloud/operator/build-workflow-images-with-tekton.adoc

```diff
@@ -0,0 +1,143 @@
+        - name: url
+          value: $(params.git-url)
+        - name: subdirectory
+          value: ''
+        - name: deleteExisting
+          value: 'true'
+        - name: revision
+          value: $(params.git-revision)
+      taskRef:
+        kind: ClusterTask
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+    - name: build-image
+      params:
+        - name: IMAGE
+          value: $(params.IMAGE)
+        - name: TLSVERIFY
+          value: 'false'
+      runAfter:
+        - fetch-repository
+      taskRef:
+        kind: ClusterTask
+        name: buildah
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  workspaces:
+    - name: shared-workspace
+----
+
+== Create The PipelineRun to execute the pipeline
+
+The next step is to create a pipeline run with the details of your git repo and the name of the image desidered
+
+in the Pipeline menu, in the item PipelineRun copy the following yaml replacing the placeholder with your values
+
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: custom-workflow-image-pipeline-run
+  namespace: kogito-serverless-operator-system
+spec:
+  workspaces:
+    - name: shared-workspace
+      volumeClaimTemplate:
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 500Mi
+  params:
+    - name: deployment-name
+      value: <your-image-name>
+    - name: git-url
+      value: https://<your repo>.git
+    - name: git-revision
+      value: "main"
+    - name: IMAGE
+      value: "image-registry.openshift-image-registry.svc:5000/kogito-serverless-operator-system/<your-image-name>:latest"
+  pipelineRef:
+    name: custom-workflow-image-pipeline
+----
+You can check the execution of the pipeline in the UI,
+at the end your image will be available at:
+image-registry.openshift-image-registry.svc:5000/kogito-serverless-operator-system/<your-image-name>:latest
+
+
+== Create The PipelineRun with Tkn cli
+
+If you have installed the tekton cli you could run the pipeline with this command
+----
+tkn pipeline start kogito-serverless-operator-pipeline \
+-w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/tekton/volume/persistent_volume.yaml \
+-p deployment-name=<your-image-name> \
+-p git-url=https://<your repo>.git \
+-p git-revision=main \
+-p IMAGE='image-registry.openshift-image-registry.svc:5000/kogito-serverless-operator-system/<your-image-name>:latest' \
+--use-param-defaults
+----
+
+This command return an id to check the execution with the cli in this way
+----
+tkn pipelinerun logs custom-workflow-image-pipeline-run-<id> -f -n <your-namespace>
+----
+
+at the end your image will be available at:
+image-registry.openshift-image-registry.svc:5000/kogito-serverless-operator-system/<your-image-name>:latest
```

Comment:
```suggestion
----
image-registry.openshift-image-registry.svc:5000/kogito-serverless-operator-system/<your-image-name>:latest
----
```

Reply:
image-registry.openshift-image-registry.svc:5000/ can vary from ocp to ocp cluster.

Reply:
ok, I'll add the explanation as example